src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+const calendar = document.querySelector('.calendar');
+
+const daysInMonth = 31;
+
+function generateCalendar(daysInMonth) {
+
+  // Add divs for each day in the month
+  for (let i = 1; i <= daysInMonth; i++) {
+    const day = document.createElement('div');
+    day.classList.add('calendar__day');
+    day.setAttribute('data-day', i);
+    calendar.appendChild(day);
```

Comment:
You are appending the day to the calendar inside the loop. It's a good practice to avoid updating the DOM in a loop because it can lead to performance issues. Instead, you can create a document fragment, append the days to this fragment in the loop, and then append the fragment to the calendar after the loop.